src/components/challenges/active.tsx

```diff
@@ -1,7 +1,8 @@
 import { FC, useState } from 'react';
-import classNames from '@/components/challenges/styles.module.scss';
-import Details from '@/components/challenges/details';
-import Participants from '@/components/challenges/participants';
+// import classNames from '@/components/challenges/styles.module.scss';
```

Comment:
@ASHWINIAJIT Please remove the all commented lines.

Reply:
Removed all commented lines.